packages/aws-cdk-lib/aws-backup/lib/rule.ts

```diff
@@ -213,6 +224,10 @@ export class BackupPlanRule {
       throw new Error('`scheduleExpression` must be of type `cron`');
     }
 
+    if (props.scheduleExpressionTimezone && !props.scheduleExpression) {
```

Comment:
there are some `scheduleExpression`s that don't make sense with a timezone, and this check ignores that.